client/stripe-utils/utils.js

```diff
@@ -23,15 +23,13 @@ import { errorTypes, errorCodes } from './constants';
  * @return  {StripeServerData} Stripe server data.
  */
 const getStripeServerData = () => {
-	// Classic checkout or blocks-based one.
-	const stripeServerData =
-		typeof wc_stripe_upe_params !== 'undefined' // eslint-disable-line camelcase
-			? wc_stripe_upe_params // eslint-disable-line camelcase
-			: wc?.wcSettings?.getSetting( 'stripe_data', null );
-	if ( ! stripeServerData ) {
+	// Classic checkout.
+	// eslint-disable-next-line camelcase
+	if ( ! wc_stripe_upe_params ) {
 		throw new Error( 'Stripe initialization data is not available' );
 	}
-	return stripeServerData;
+	// eslint-disable-next-line camelcase
+	return wc_stripe_upe_params;
 };
```

Comment:
This function is now exclusively used for the UPE checkout, since it was causing a conflict with the Blocks integration. See the commit message in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1825/commits/a9774f9f6745f8effaa9d3fb0df6d133248999fa for more details.